Client/mods/deathmatch/logic/luadefs/CLuaObjectDefs.cpp

```diff
@@ -38,6 +39,13 @@ void CLuaObjectDefs::LoadFunctions()
         {"toggleObjectRespawn", ToggleObjectRespawn},
         {"setObjectMass", SetObjectMass},
         {"setObjectProperty", SetObjectProperty},
+
+        // Object util functions
+        {"getObjectLowLODModel", ArgumentParser<GetObjectLowLODModel>},
+        {"getObjectHighLODModel", ArgumentParser<GetObjectHighLODModel>},
+        {"setObjectCustomLowLODModel", ArgumentParser<SetObjectCustomLowLODModel>},
+        {"resetObjectCustomLowLODModel", ArgumentParser<ResetObjectCustomLowLODModel>},
+        {"resetAllObjectCustomLowLODModels", ArgumentParser<ResetAllObjectCustomLowLODModels>},
```

Comment:
Maybe
`getModelLowLOD`
`getModelHighLODModel`
...

Reply:
i disagree, this notation is the clearest

get Object Low Level of Detail Model (high level of detail model)
get Object High Level of Detail Model (low level of detail model)
...

Reply:
But this function works with buildings models too.

Reply:
> But this function works with buildings models too.

Buildings use object models.

Reply:
There is no such thing as object models too.  The game engine uses different model types like atomic, clump, timed, vehicle, ped, etc. 
getObjectLowLODModel says me that this function works with object elements.
For vehicles, we are using `getVehicleModel*` function names.

Reply:
Object models is a thing. = Object IDs

We don't use any function getVehicleModel...

I could rename to `getLowLODObjectModel` ?

Reply:
> We don't use any function getVehicleModel...


[getVehicleModelDummyDefaultPosition](https://wiki.multitheftauto.com/wiki/GetVehicleModelDummyDefaultPosition)
[getVehicleModelDummyPosition](https://wiki.multitheftauto.com/wiki/GetVehicleModelDummyPosition)
[getVehicleModelExhaustFumesPosition](https://wiki.multitheftauto.com/wiki/GetVehicleModelExhaustFumesPosition)
[getVehicleModelWheelSize](https://wiki.multitheftauto.com/wiki/GetVehicleModelWheelSize)

Reply:
objects have no lod support in SA. MTA uses custom implementation for them.

Reply:
> objects have no lod support in SA. MTA uses custom implementation for them.

What are you implying?